addons/mass_mailing/models/mailing.py

```diff
@@ -220,7 +220,6 @@ def _compute_statistics(self):
             total = row['expected'] = (row['expected'] - row['ignored']) or 1
             row['received_ratio'] = 100.0 * row['delivered'] / total
             row['opened_ratio'] = 100.0 * row['opened'] / total
-            row['clicks_ratio'] = 100.0 * row['clicked'] / total
```

Comment:
The `_compute_statistics` method is not the compute method of the `clicks_ratio` field. This was leading the serialisation errors as one write on a non-protected field during a read()

Reply:
Maybe this should be done in a separate commit (and backported ?)

Reply:
Done ! Two commits here plus a backport ready in v12 #67043